agent/config/config.go

```diff
@@ -474,6 +474,7 @@ func environmentConfig() (Config, error) {
 		ImageCleanupInterval:               parseEnvVariableDuration("ECS_IMAGE_CLEANUP_INTERVAL"),
 		NumImagesToDeletePerCycle:          parseNumImagesToDeletePerCycle(),
 		ImagePullBehavior:                  parseImagePullBehavior(),
+		ImageNotDeleteList:                 parseImageNotDeleteList("ECS_IMAGES_NOT_DELETE"),
```

Comment:
What about `ECS_IMAGE_CLEANUP_EXCLUDE` and `ImageCleanupExclusionList`?  I think that's more clear.

Reply:
yeah, will change it.